content/en/docs/reference/glossary/container-runtime-interface.md

```diff
@@ -11,12 +11,12 @@ tags:
   - cri
 ---
 
-The main protocol for the communication between the kubelet and Container Runtime.
+The Container Runtime Interface (CRI) is the main protocol for the communication
+between the kubelet and Container Runtime.
```

Comment:
The original is better. If we want to include the glossary definition into a page, but prefixed with some text, we have a [mechanism](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#glossary) for that.

Reply:
Sounds good, let me give that a try.